app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -244,7 +244,7 @@ define([
         ],
         'stripped-min-length': [
             function (value, param) {
-                return $(value).text().length >= param;
+                return value.length >= param;
```

Comment:
As a rule has `stripped-min-length` name, value should be stripped (use `utils.stripHtml`).

A simple check on `>= param` will make any field with this validation rule required so we should allow empty values. To avoid this implementation should be aligned with implementation of `min_text_length` rule.

Reply:
"As a rule has stripped-min-length name, value should be stripped (use utils.stripHtml)." OK

"A simple check on >= param will make any field with this validation rule required. To avoid this implementation should be aligned with implementation of min_text_length rule." When you say "should be aligned" I should validate the values like min_text_length (return _.isUndefined(value) || value.length === 0 || utils.stripHtml(value) >= params;) or I should use min_text_lenght function?

Reply:
Function reuse is preferred as  it reduces code duplication but as there is no much code it is ok just to make same check inside stripped-min-length